tests/rbd_mirror/test_rbd_mirror_reconfig.py

```diff
@@ -0,0 +1,113 @@
+from tests.rbd_mirror.rbd_mirror_utils import rbd_mirror_config
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def test_mirror_reconfig(rbd_mirror, pool_type, **kw):
+    """Method to validate reconfiguring mirror on existing mirror cluster
+    got prevented or not.
+
+    Args:
+        rbd_mirror: Object
+        pool_type: Replicated or EC pool
+        **kw: test data
+
+    Returns:
+        int: The return value. 0 for success, 1 otherwise
+    """
+    try:
+        config = kw.get("config")
+        mirror1 = rbd_mirror.get("mirror1")
+        mirror2 = rbd_mirror.get("mirror2")
+        pool = config[pool_type]["pool"]
+        mode = config[pool_type]["mode"]
+        primary_cluster = kw.get("ceph_cluster")
+        primary_cluster_name = primary_cluster.name
+        mirror1.bootstrap_peers(poolname=pool, cluster_name=primary_cluster_name)
+        # Attempt creating mirror in same cluster this should be prevented
+        out, err = mirror1.import_bootstrap(
+            poolname=pool, cluster_name=primary_cluster_name, all=True, check_ec=False
+        )
+        if "cannot import token for local cluster" in err:
+            log.info("importing bootstrap token with same cluster got prevented")
+        else:
+            log.error("importing bootstrap token with same cluster not prevented")
+            return 1
+
+        out = mirror1.enable_mirroring(mirror_level="pool", specs=pool, mode=mode)
+        if "mirroring is already configured" in out:
+            log.info(f"creating mirror with same cluster got prevented for {mode} mode")
+        else:
+            log.error(
+                f"creating mirror with same cluster not prevented for {mode} based"
+            )
+            return 1
```

Comment:
I don't see much value addition here from the perspective of testing prevention of configuring mirroring with same cluster.

pool enable - this command would fail irrespective of same cluster or different cluster.

Reply:
agree but it would be good to cover that scenario as well right as coding is already done for it 
there may be chance that enabling mirroring to already enabling mirror command might pass if there is a bug
hence to cover that scenario as well,
Kept the remaining code as it is and covered the scenario for the mirroring bootstrap to fail.

Reply:
Discussed in automation peer review meeting and agreed to keep the code with modifications to the polarion test case.

Please modify the TC in polarion and request for review.

Reply:
Updated polarion TC please review it.